### ISSUE TITLE
[Feature] : time boxing

### ISSUE THREAD

**Issue by imed-ghomari** [grassbl8d/flexible-pomo-obsidian#1]
I was wondering if the features available in this [mac app](https://www.effortless.app/) or this [android app](https://play.google.com/store/apps/details?id=com.slamtastic.donow.app&hl=en&gl=US) or this [ios app](https://www.producthunt.com/posts/time-2-2) could be implemented. Given that, timeboxing is similar to pomodoro as the two have countdown timers.

The features I'm interested in are :
- To manually set duration for each tasks with the option of not using breaks in between.
- The time when I'll finish the task based on the duration set (automatically generated)
- Notification when the duration time is finished

There is a [similar feature request](https://github.com/lynchjames/obsidian-day-planner/issues/109) in the [day planner](https://github.com/lynchjames/obsidian-day-planner) plugin, but there is no active development for it, so I thought maybe this plugin could do it.

**Comment by fjovqiybue846** [comment]
Maybe you can use the dataview to calculate the behavior in the log. Use a template to automate statistics once a day

**Comment by fjovqiybue846** [comment]
Of course, I think if this plugin can set the total duration of the task, and automatically count it. That would be a great feature. I thought maybe we could make a label. Something like a task.
